### PR TITLE
feat(BraniTube): renamed to Aniyan

### PR DESCRIPTION
**Pull request by TheDropX**
**Describe the changes in this pull request:**

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

Discord_exUpmEDZh0
Discord_jocVkMfsPW

</details>

### CODE REVIEW

File: websites/A/Aniyan/presence.ts [hunk 1]
```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
 
 let lastPlaybackState = null,
   playback,
-  browsingTimestamp = Math.floor(Date.now() / 1000);
+  browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:11]: no pref is former version

File: websites/A/Aniyan/presence.ts [hunk 2]
```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
...
 
 if (lastPlaybackState !== playback) {
   lastPlaybackState = playback;
-  browsingTimestamp = Math.floor(Date.now() / 1000);
+  browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:15]: no pref is former version

File: websites/A/Aniyan/presence.ts [hunk 3]
```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
...
   }
 
-  const video: HTMLVideoElement = document.querySelector(
-    "#player > div.jw-media.jw-reset > video"
-  );
-
-  if (video) {
-    const videoTitle = document.querySelector(
+  if (playback) {
+    const videoTitle: HTMLElement = document.querySelector(
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:39]: pref
```suggestion
    const videoTitle = document.querySelector<HTMLDivElement>(
```

File: websites/A/Aniyan/presence.ts [hunk 4]
```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
...
         "div > div.episodeInfo > div.nomeAnime"
-      ) as HTMLElement,
-      episode = document.querySelector(
+      ),
+      episode: HTMLElement = document.querySelector(
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:42]: pref
```suggestion
      episode = document.querySelector<HTMLDivElement>(
```

File: websites/A/Aniyan/presence.ts [hunk 5]
```diff
@@ -72,3 +70,16 @@ presence.on("UpdateData", async () => {
...
+ * @param {Number} videoTime Current video time seconds
+ * @param {Number} videoDuration Video duration seconds
+ */
+function getTimestamps(videoTime: number, videoDuration: number): number[] {
+  const startTime = Date.now();
+  return [
+    Math.floor(startTime / 1000),
+    Math.floor(startTime / 1000) - videoTime + videoDuration
+  ];
+}
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:85]: not used
```suggestion
```

File: websites/A/Aniyan/presence.ts [hunk 6]
```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
...
 presence.on("UpdateData", async () => {
-  playback =
-    document.querySelector("#player > div.jw-media.jw-reset > video") !== null
-      ? true
-      : false;
+  const video: HTMLVideoElement = document.querySelector(
+    "#player > div.jw-media.jw-reset > video"
+  );
+
+  playback = video !== null ? true : false;
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:23]:
```suggestion
  playback = video ? true : false;
```

File: websites/A/Aniyan/presence.ts [hunk 7]
```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
...
-  browsingTimestamp = Math.floor(Date.now() / 1000);
+  browsingStamp = Math.floor(Date.now() / 1000);
 }
 
 presence.on("UpdateData", async () => {
-  playback =
-    document.querySelector("#player > div.jw-media.jw-reset > video") !== null
-      ? true
-      : false;
+  const video: HTMLVideoElement = document.querySelector(
```
Comment (EncryptedDev) [inline websites/A/Aniyan/presence.ts:19]:
```suggestion
  const video = document.querySelector<HTMLVideoElement>(
```

**Review by EncryptedDev — DISMISSED** [review]

File: websites/A/Aniyan/presence.ts (hunk 6 above)
Comment (TheDropX) [reply websites/A/Aniyan/presence.ts:23]: the playback variable is used somewhere in the top part of the code, I am not removing it.

**Review by Bas950 — DISMISSED** [review]
fix conflicts

**Review by EncryptedDev — DISMISSED** [review]
add schema

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**